crypto/bn/bn_lcl.h

```diff
@@ -367,8 +367,6 @@ struct bn_gencb_st {
  * "bignum" routines (and have the compiler optimize 'em)
  * exhibiting "native" performance in C. That's what BN_UMULT_HIGH
  * macro is about:-)
- *
- *                                      <appro@fy.chalmers.se>
```

Comment:
The above commentary is also "personal" and a bit out-of-date. First of all omit beginning of commentary up to [but not including] "if the compiler doesn't support 2\*N integer type". And then at the end add something like "though more recent compilers do support 2\*64 integer type, which is also used here."